contributing/documentation/content_guidelines.rst

```diff
@@ -160,6 +160,30 @@ A few tips to improve your screenshots:
    Resizing the window's width is the most important step to do as Odoo's responsive design
    automatically resizes all fields to match the window's width.
 
+.. _contributing/media-filenames:
+
+Media filenames
+---------------
+
+A **media filename** is written in **lower-case letters** and is composed of three parts:
+
+1. A **prefix** which is the same name as the *RST filename* in which the media is referenced. E.g.,
+   ``content-guidelines``.
```

Comment:
I'm not sure that requiring to prefix the image filenames with the page name is a good practice. The context is already provided by the page itself, and it is generally recommended to describe the image in its filename, not so much the page theme.
Furthermore, Google recommends using the same keywords (as a rule of thumb) in `alt` texts as in image filenames.
Also, image filenames should comprise too many words. Start from a long *page* name and you're already screwed.

Reply:
it's to keep a form of organization in the folder.
I understand your point of view, but from a writer's perspective, it nearly never happens that we use an image in two different documents. Also, it can become a problem for later updates as the media's usage in various docs isn't always clear to the person modifying/deleting it. 

alternatively, having a media folder per page would be another way to keep the media files organized, I guess.

about the alt thing, do you want me to add a comment about this both in the filename and alt tags sections?

Reply:
> alternatively, having a media folder per page would be another way to keep the media files organized, I guess.

Yeah, or even a folder per page inside the media folder.
That means longer references in RST but at least it doesn't hurt the SEO nor the clarity of the filename.

Reply:
up to you! But I think it'd be better organized indeed this way, indeed.
Also, soooo much easier to compress with `pngquant *` for all files at once :p

Reply:
If really image files of two pages have nothing in common, you won't probably create them all in the same PR anyway :p

Reply:
So your it would be something like this, with one media folder and sub folders:

```
├── section
│   └── media
│   │   └── doc1
│   │   │   └── something.png
│   │   │   └── something-else.png
│   └── doc1.rst

```
or multiple media folders:

```
├── section
│   └── doc1-media
│   │   └── something.png
│   │   └── something-else.png
│   └── doc1.rst
```

Reply:
Either are fine, a single media folder is too IMO. Simply pick the best fit for your use-case.

Reply:
A single media folder is fine, but then we come full circle ^^ :
 I would add a prefix in the filename to organize the files. That's how I do it currently.

Reply:
I prefer the "one media folder and sub folders" option.
Either way, I think we should impose a single way to organize images. It makes navigating folders easier than if several ways are used - and I don't see any benefit in using multiple ways.

Reply:
I'm rewriting this with the option 2 in mind (but without the `-media` after). So that would mean one .rst file + it's folder (same name)

Reply:
That's a bad idea IMO.
This structure is already used for RST indexes. The `sales.rst` file references files inside `sales/` directory.
If you use it for media too, we'll look into the folder and find images instead of RST.

Reply:
I gave it a second thought and it doesn't sound that bad actually. It's surely confusing from a technical perspective but I guess that we'll get used to it.

Another good argument in favor of the individual media folder named after the referencing RST file is the (future) new structure of the doc. After the doc apocalypse, we'll have to move content up in the hierarchy: introduction pages' content should be written in toctree pages rather than buried deep in the tree of contents.
It means that RST files that already have their homonym directory will start referencing images. Those images could then be added in those homonym directories alongside the referenced RST files (sub-pages). The folders won't be media-only folders, nor doc-only, but will start acting as source folders which is just fine.